src/Model/ItemVersion.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\Model;
+
+use function fseek;
+use function fstat;
+use function ftell;
+use function hash_final;
+use function hash_init;
+use function hash_update_stream;
+use function is_int;
+
+final class ItemVersion
```

Comment:
Class is final and with no interface, which means it's irreplaceable; but `fstat`, `fseek` etc are assumptions of a local filesystem so this looks more like a specific implementation of `ItemVersion` than a generic one that can use multiple backends

Comparing with Postgres, I expect database drivers to have their own interfaces for retrieving remote content: https://secure.php.net/manual/en/function.pg-lo-seek.php

Reply:
Seems they both work with PDO LOB, but yeah, it needs to be more portable. (Not sure if setting `Content-Length` is useful anyway, so can probably ignore the size.)